internal/weldr/api.go

```diff
@@ -163,24 +159,28 @@ func New(repoPaths []string, stateDir string, solver *dnfjson.BaseSolver, dr *di
 	}
 
 	var hostArch distro.Arch
-	hostDistro := dr.GetDistro(hostDistroName)
+	hostDistro := dr.FromHost()
+	hostDistroName := ""
```

Comment:
No need to define ` hostDistroName` variable here, since it is used only in the `else {` branch of the `if` statement below.

Reply:
When I was going commit after commit, this was not the case. But with the all changes applied, you are right.